docs/setup.md

```diff
@@ -31,3 +31,10 @@ asdf install djinni latest
 
 Download the [latest released batch file](https://github.com/cross-language-cpp/djinni-generator/releases/latest/download/djinni.bat), copy it where you want and [put the folder on your PATH](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/).
 
+### Conan
+
+The generator is available at [conan-center](https://conan.io/center/djinni-generator) for Windows, Linux & macOS:
+
+```sh
+conan install djinni-generator/0.3.1@
+```
```

Comment:
I've added the conan install instructions twice. Once here in the docs and once in the Readme for github visitors.